Dockerfile.fakemachine-arch

```diff
@@ -1,11 +1,44 @@
+# Build unpackaged dependencies from source
+FROM archlinux:base-devel AS builder
+
+# Build user-mode linux from AUR
+ENV EUID=1
+WORKDIR /tmp
+RUN pacman -Syu --noconfirm base-devel \
+                            bc \
+                            git \
+                            inetutils \
+                            libffi && \
+    gpg --recv-keys 38DBBDC86092693E && \
+    git clone --depth=1 https://aur.archlinux.org/linux-usermode.git && \
+    cd linux-usermode && \
+    export MAKEFLAGS="-j$(nproc)" && \
+    makepkg -s
```

Comment:
I would omit base-devel (already installed, what's why the container is called base-devel), inetutils and bc (already installed with makepkg -s). The git is obvious (git clone just below), but the libffi isn't - please add a note or remove. 

Since we don't check the gpg key, i would drop the gpg command and use `makepkg --skippgpcheck ...`.

Reply:
thanks, i made makepkg download dependenceis also :-)